index.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+     <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!-- Fonts -->
+    <link rel="stylesheet" href="https://use.typekit.net/tyg1nlb.css">
+    <title>WLIT Hackathon 2021</title>
+</head>
+<body>
+    <main>
+            <section class="up-header">
+                <div class="container-fluid">
+                    <div class="container nav-top">
+                        <nav class="navbar d-flex justify-content-end">
+                            <div class="content">
+                                <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+                                <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+                                <a href="#">English</a>
+                                <a href="#"> My page</a>
+                                <a href=""> Logout </a>
+                            </div>
+                        </nav>
+                    </div>
+                </div>
+            </section>   
+
+            <section class="hamburger-page">
+                
+                    <nav class="navbar navbar-expand-md navbar-md-light bg-light">
+                        <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                            <a href="#">
+                                <img src="./images/close.png" id="close-ham-img" alt="closeX">
+                            </a>
+                            <a href="#" class="navbar-brand">Laxmi</a>
+                            
+                            <ul class="navbar-nav ms-auto">
+                                <li class="nav-item">
+                                    <a class="nav-link active" aria-current="page" href="./pages/about.html">About</a>
```

Comment:
- [ ] This link leads to no page in your project. This happens when the home page is open in mobile view and the user clicks on the About link in the mobile menu. Refactor it to:
```suggestion
<a class="nav-link active" aria-current="page" href="about.html">About</a>
```